api/metrics/v1/rules.go

```diff
@@ -0,0 +1,67 @@
+package v1
+
+import (
+	"io"
+	"net/http"
+
+	"github.com/observatorium/api/authentication"
+	"github.com/observatorium/api/rules"
+)
+
+type rulesHandler struct {
+	client *rules.Client
+}
+
+func (rh *rulesHandler) get(w http.ResponseWriter, r *http.Request) {
+	tenant, ok := authentication.GetTenant(r.Context())
+	if !ok {
+		http.Error(w, "error finding tenant", http.StatusUnauthorized)
+		return
+	}
+
+	resp, err := rh.client.ListRules(r.Context(), tenant)
+	if err != nil {
+		http.Error(w, "error listing rules %w", http.StatusInternalServerError)
```

Comment:
Maybe it makes sense to propagate the HTTP status code we got from the rules backend instead of `500`. I am thinking about the cases when it fails some validation checks on Rules Backend side and it returns a `400 Bad Request`.